.husky/pre-commit

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+FILES=$(git diff --cached --name-only --diff-filter=ACMR | sed 's| |\\ |g')
+[ -z "$FILES" ] && exit 0
+
+# Run lint-staged on all staged
+echo "$FILES" | xargs node_modules/.bin/lint-staged
```

Comment:
Probably better to invoke it as a yarn command, right?

```suggestion
echo "$FILES" | yarn lint-staged

```

(added a new line which was missing)